src/utils/navigation.ts

```diff
@@ -208,22 +208,22 @@ export const NAVIGATION: Navigation = {
   [config.BENEFICIAL_OWNER_TYPE_URL]: {
     currentPage: config.BENEFICIAL_OWNER_TYPE_PAGE,
     previousPage: () => config.BENEFICIAL_OWNER_STATEMENTS_URL,
-    nextPage: [config.CHECK_YOUR_ANSWERS_URL, config.TRUST_INFO_URL]
+    nextPage: [config.CHECK_YOUR_ANSWERS_PAGE, config.TRUST_INFO_URL]
```

Comment:
What's the reason why this has changed from URL to PAGE? Looks like a potential breaking change. Have you run it locally? Same with line 221.

Reply:
yes, they should be URL as the PAGE is just a template name.

Reply:
edited back to url. As for AC3 it may have to be a separate story as it relates to the same change links as on AC2

Reply:
the change still shows as config.CHECK_YOUR_ANSWERS_PAGE, did you push up your changes as github says last commit was 4 days ago

Reply:
pushed up now